src/backend/distributed/commands/multi_copy.c

```diff
@@ -199,6 +200,9 @@ CitusCopyFrom(CopyStmt *copyStatement, char *completionTag)
 		Oid relationId = RangeVarGetRelid(copyStatement->relation, NoLock, false);
 		char partitionMethod = PartitionMethod(relationId);
 
+		/* disallow modifications to a partition table which have rep. factpr > 1 */
+		EnsurePartitionTableNotReplicated(relationId);
```

Comment:
This is a major restriction. It will render whole feature useless if we disallow copy.

Reply:
Can't you still copy into the parent?

Reply:
Correct,  I take it back.

Reply:
Only downside would be users won't be able to insert data into `cstore_fdw` partitions  in PG10.